README.md

```diff
@@ -82,6 +125,50 @@ Returns a rejected bluebird promise with the given value.
 
 *Note: If the given value is a String, that string will be wrapped in an Error object and will be on the message property.*
 
+### Spies
+
+#### Return Value Methods
+
+*Promises cannot be in a pending state otherwise an error will be thrown.*
+
+**.returnedPromise(value)**
+
+Identical to sinon `.returned()` except it automatically unwraps the bluebird promise(if a promise is returned) and compares directly to the unwrapped value.
+
+**.alwaysReturnedPromise(value)**
+
+Identical to sinon `.alwaysReturned()` except it automatically unwraps the bluebird promise(if a promise is returned) and compares directly to the unwrapped value.
+
+#### Called With Methods
+
+*The promise (or promises) can be anywhere in the list of arguments passed into the spied on function or none at all.*
+
+*Promises cannot be in a pending state otherwise an error will be thrown.*
+
+**.calledWithPromise(value, ...)**
+
+Identical to sinon `.calledWith()` except it automatically unwraps the bluebird promise(if a promise is passed in) and compares directly to the unwrapped value.
+
+**.calledWithMatchPromise(value, ...)**
+
+Identical to sinon `.calledWithMatch()` except it automatically unwraps the bluebird promise(if a promise is passed in) and compares directly to the unwrapped value.
+
+**.alwaysCalledWithPromise(value, ...)**
+
+Identical to sinon `.alwaysCalledWith()` except it automatically unwraps the bluebird promise(if a promise is passed in) and compares directly to the unwrapped value.
+
+**.alwaysCalledWithMatchPromise(value, ...)**
+
+Identical to sinon `.alwaysCalledWithMatch()` except it automatically unwraps the bluebird promise(if a promise is passed in) and compares directly to the unwrapped value.
+
+**.calledWithExactlyPromise(value, ...)**
+
+Identical to sinon `.calledWithExactly()` except it automatically unwraps the bluebird promise(if a promise is passed in) and compares directly to the unwrapped value.
+
+**.alwaysCalledWithExactlyPromise(value, ...)**
+
+Identical to sinon `.alwaysCalledWithExactly()` except it automatically unwraps the bluebird promise(if a promise is passed in) and compares directly to the unwrapped value.
+
```

Comment:
This is all good -- I would just consider making the "Identical to sinon ..." disclaimer a generic explanation that we're simply appending "Promise" to each one and move that to the top of this section. That way you can simply list the applicable methods as a bullet list -- that would communicate the same thing more succinctly.